Sources/Tracking/Store/SdkConfig.swift

```diff
@@ -44,6 +44,11 @@ public struct SdkConfig {
      */
     public var autoScreenViewBody: (() -> [String: Any])?
 
+    /**
+     Enable this property if you want SDK to automatic tracking of device attributes such as
+     operating system, device locale, device model, app version etc
+     */
+    public var autoTrackDeviceAttributes: Bool = true
```

Comment:
1. Our squad decided to have the default value to be `false` like we do with screen view tracking. We want to avoid automatically enabling data collection and allow customers to decide when they are used. 

2. In the squad sync call a couple days ago, we mentioned that instead of having just 1 property for enabling/disabling default device attributes we have 1 property for every type of device attribute. So, something like this:
```swift
CustomerIO.config { 
  $0.deviceAttributeOSVersion = true
  $0.deviceAttributeDeviceModel = true
  ...
}
```

I am unsure about the naming to use? Maybe we do this instead:

```swift
CustomerIO.config { 
  $0.enableDefaultDeviceAttributes = [OSVersion, DeviceModel]
}
```

Also be sure to add to the [APITest](https://github.com/customerio/customerio-ios/blob/0ca06183f9c0094019e3e58d0963b5931909c348/Tests/Tracking/APITest.swift#L87-L95)

Reply:
Yes, we have to implement it in such a way that each property is configurable. But as far as I remember, in sync call I concluded by saying that right now the implementation has a group enable/disable and can be changed. I am planning to do this in a new PR and not block this one for the purpose ! What do you think ?

Reply:
Tagging you @levibostian so that you can find this suggestion easily and share your thoughts.

+

Default should be `false` Thanks for pointing out

Reply:
https://github.com/customerio/issues/issues/6510#issuecomment-1063800680